praktikumsplaner-frontend/frontend/src/components/praktikumsplaetze/Praktikumsplaetze/PraktikumsstelleCard.vue

```diff
@@ -64,17 +74,34 @@
     </v-container>
 </template>
 <script setup lang="ts">
-import { ref } from "vue";
+import { computed, ref } from "vue";
 
 import PraktikumsstellenService from "@/api/PraktikumsstellenService";
 import YesNoDialogWithoutActivator from "@/components/common/YesNoDialogWithoutActivator.vue";
+import AusbildungsPraktikumsstelleUpdateDialog from "@/components/praktikumsplaetze/Praktikumsplaetze/AusbildungsPraktikumsstelleUpdateDialog.vue";
+import StudiumsPraktikumsstelleUpdateDialog from "@/components/praktikumsplaetze/Praktikumsplaetze/StudiumsPraktikumsstelleUpdateDialog.vue";
 import { useTextGenerator } from "@/composables/textGenerator";
 import emitter from "@/stores/eventBus";
 import Praktikumsstelle from "@/types/Praktikumsstelle";
 
 const properties = defineProps<{
     modelValue: Praktikumsstelle;
 }>();
+const emits = defineEmits<{
+    (e: "updated", praktikumsstelleToUpdate: Praktikumsstelle): void;
```

Comment:
I think for the v-model directive to work it should be "update:modelValue"

Reply:
In https://github.com/it-at-m/Praktikumsplaner/blob/feature/updatePraktikumsstellen/praktikumsplaner-frontend/frontend/src/components/praktikumsplaetze/Praktikumsplaetze/PraktikumsstellenList.vue the model-value binding should be changed to v-model directive